docs/vexim-acl-check-rcpt.conf

```diff
@@ -52,5 +52,16 @@
 #                          and spamassassin='1'}}}{1} {yes}{no}}
 #          !acl          = spf_rcpt_acl
 
+#
+# To use a DNSBL the best would be to set up a caching dns server for you own
+# otherwise you might get rate limited and your requests are not processed
+# 
+# Also disable spamchecking for authenticated users because dial up networks
+# are often marked as bad in case of sending mails
+#
+  accept
+          authenticated = *
+          condition = ${if eq{$authenticated_id}{$sender_address}{yes}{no}}
```

Comment:
Yes, currently the check_rcpt-section is much longer and this file is only a small part:
https://github.com/vexim/vexim2/blob/master/docs/configure#L339-L419

We could move the include-command for this file behind line 364 (mails to postmaster are accepted even if the host is on a blacklist, and invalid local-parts are very fast compared to DNS lookups). Then we can move the accept-authenticated-rule (line 414) into this file. And remove lines 369-381.

Should I prepare a PR?